config/nightwatch-globals-sauce.js

```diff
@@ -0,0 +1,30 @@
+module.exports = {
```

Comment:
This is somewhere between configuration and executable code... prior to this change the configuration all went in the config directory, so I've stuck with that pattern. The nightwatch-commands directory is a bit different, since they're extensions to the testing library, so even if we could split out the executable part of this I'm not sure it would belong there.